packages/core/src/components/forms/inputGroup.tsx

```diff
@@ -40,6 +40,9 @@ export interface InputGroupProps
     /** Whether this input should use large styles. */
     large?: boolean;
 
+    /** The callback invoked when the value changes.
+    onValueChange?(valueAsString: string/*, inputElement: HTMLInputElement | null*/): void;
```

Comment:
I think there are some commenting issues here

Reply:
thanks Andrew. left this here as a TODO. https://github.com/palantir/blueprint/pull/6290/files#r1268590322